scheduler/src/main/resources/application.properties

```diff
@@ -12,3 +12,11 @@ scalingengine.notification.reschedule.maxcount=3
 
 # scaling engine url
 autoscaler.scalingengine.url=http://test-scalingengine-test
+
+#ssl
+server.ssl.key-store=../test-certs/scheduler.jks
+caCert=../test-certs/autoscaler-ca.crt
+server.ssl.key-alias=scheduler
+server.ssl.key-store-password=123456
+#server.ssl.key-password=123456
+server.ssl.key-store-type=JKS
```

Comment:
why not just use pkcs12? makes it easier for everyone. which is what you generated from the tooling.